turbinia/notify.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from turbinia import config
+
+#Sends notfications via email
+def sendmail(message):
+
+    log=logging.getLogger('turbinia')
+    try:
+        EMAIL_NOTIFICATIONS = config.EMAIL_NOTIFCATIONS
+        if EMAIL_NOTIFICATIONS == 'true':
+            EMAIL_HOST_ADDRESS = config.EMAIL_HOST_ADDRESS
+            EMAIL_PORT = int(config.EMAIL_PORT)
+
+            EMAIL_ADDRESS = config.EMAIL_ADDRESS
+            EMAIL_PASSWORD = config.EMAIL_PASSWORD
+
+            RECIEVING_ADDRESS = config.RECIEVING_ADDRESS
+            #Imports SMTP and  MIME(The format used for emails)
```

Comment:
Why import these here and not at the top of the file?  Also, can you add them to the requirements.txt?